### ISSUE TITLE
Podcast external links are missing visually hidden "external" text

### ISSUE THREAD

**Issue by jroebu14** [bbc/simorgh#9387]
**Describe the bug**
Podcast external links are missing an aria-label or visually hidden text with the external links translation. This violates the following acceptance criteria for other external links.

> As a WS screenreader user
> I want external links to be identified as in my language
> so that I know if I click on the link I'll be taken from the BBC website

Should we fix this for the podcast external links?

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.bbc.com/persian/podcasts/p02pc9wf
2. With devtools, inspect an external link e.g. Spotify
3. Observe it is missing visually hidden text indicating that the user will be taken to an external page outside of the BBC website.

**Expected behaviour**
It should have visually hidden text ", external" translated to the correct language for the service.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**
This should have an developer accessibility swarm.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

**Comment by jroebu14** [comment]
@greenc05 do you have any thoughts on this one? Thanks

**Comment by greenc05** [comment]
Good spot though happy to not have this in this instance unless others feel strongly? On a article page it's harder to tell if a link will be external due to the body text that could be used for the link, where here it's pretty obvious as they always contain a brand name.

**Comment by greenc05** [comment]
Though I just saw that @nicola-d-bbc has done the work already, certainly no harm in having it :)

**Comment by LilyL0u** [comment]
'External' text is visible for Spotify, Apple and RSS

<img width="804" alt="Screenshot 2021-09-08 at 19 04 00" src="https://user-images.githubusercontent.com/54575620/132561429-85cfcef9-f1eb-4660-9bc9-7a8530ba1f11.png">

I understand that the removal of the translation for the word 'link' is in the next PR
